python/dgl/graph.py

```diff
@@ -4120,8 +4120,8 @@ def batch(graph_list, node_attrs=ALL, edge_attrs=ALL):
     --------
     unbatch
     """
-    if len(graph_list) == 1:
-        return graph_list[0]
+    #if len(graph_list) == 1:
```

Comment:
The test could not pass if I uncomment these two lines. @jermainewang do you have any ideas why this is the case?